.github/release-drafter.yml

```
@@ -7,7 +7,7 @@ tag-template: jenkins-$NEXT_MINOR_VERSION
 
 template: |
   _This is an automatically generated changelog draft for Jenkins weekly releases.
-  See https://www.jenkins.io/changelog/ for the official changelogs._
+  See https://www.jenkins.io/changelog/#$NEXT_MINOR_VERSION for the official changelogs._
```

```suggestion
  See https://www.jenkins.io/changelog/#v$NEXT_MINOR_VERSION for the official changelog for this release._
```

The link now goes to a specific entry (sort of), so perhaps change wording.

Also, we don't use the `v` prefix for releases, so I assume this needs to be added manually?

Do we care that this works only for the latest 30 or so releases before entries are moved to https://www.jenkins.io/changelog-old/ ?

```suggestion
  See https://www.jenkins.io/changelog/#v$NEXT_MINOR_VERSION for the official changelog for this release, or https://www.jenkins.io/changelog-old/#v$NEXT_MINOR_VERSION for releases older than around 7 months._
```

---

> Do we care that this works only for the latest 30 or so

That's already more than I'd expect someone to use GH releases over complete changelogs.